modules/custom-status/custom-status.php

```diff
@@ -414,9 +414,9 @@ function post_admin_header() {
 			// Load the custom statuses
 			foreach( $custom_statuses as $status ) {
 				$all_statuses[] = array(
-					'name' => esc_js( $status->name ),
-					'slug' => esc_js( $status->slug ),
-					'description' => esc_js( $status->description ),
+					'name' => esc_html__( $status->name, 'edit-flow' ),
```

Comment:
Could you, please, provide more details on why the `esc_js` is being swapped for `esc_html__`. Those two functions are meant for escaping in different context. As the escaping functions are being changed in terms of a changeset which claims to be fixing localisation issues, it might be helpful to have more data on why it's safe to do so.